### ISSUE TITLE
SGDet model can't  train

### ISSUE THREAD

**Issue by Lxy811** [KaihuaTang/Scene-Graph-Benchmark.pytorch#210]
Hello, when I was conducting SGDet training, I was either stuck in the training loss or when the verification results were loaded in the model verification. Could you please see that these are two kinds of errors? Is there any suggestion to solve these problems? Two Gpus 3090 are always full of usage.
![Snipaste_2024-04-13_17-16-52](https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch/assets/85209124/acc289e8-2ae6-4546-8e0a-56bfb9029e56)
![Snipaste_2024-04-13_17-17-22](https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch/assets/85209124/440be6a1-164c-4bda-86ca-ef763b9bc244)
![Snipaste_2024-04-13_17-17-35](https://github.com/KaihuaTang/Scene-Graph-Benchmark.pytorch/assets/85209124/7d3e9e66-bfad-4879-bbf0-e7b2c2b158a1)
